### DISCUSSION TITLE
string with HTML tags is not rendred

### DISCUSSION THREAD

**Discussion by marcelthiel** [a-h/templ#946] [Q&A] [answered]
Hi all,

I have a problem with one concept and I cannot find answer neither in docs nor in other sources. I have a string variable:

`text := "ABCD<em>EFGH</em>IJK"`

If I want to put it in the table like that:

`<td> { text } </td>`

The result in the browser is:
`ABCD<em>EFGH</em>IJK`

The question is: How to make those <em> </em> tags to be rendered.

**Comment by joerdav** [comment, marked as answer]
This is a safety feature described here: https://templ.guide/syntax-and-usage/expressions/#escaping

It can be circumvented with the `Raw` component: https://templ.guide/syntax-and-usage/rendering-raw-html
